### PR TITLE
[rtextures] add MixColors. a function to mix 2 colors together

### PR DESCRIPTION
**Pull request by SusgUY446**
The functions takes 2 Colors and a Float t as input. The float t is used to set what color is more dominant. 
t=0.0f - color1 is more dominant
t=0.5f - both colors are equally dominant (best choice from my testing)
t=1.0f - color2 is more dominant

### CODE REVIEW

**Comment by raysan5** [comment]
@SusgUY446 @veins1 It could be added as `ColorLerp()` in the `Color*()` functions section, seems more appropiate

**Comment by SusgUY446** [comment]
why is a Color* more appropiate when most functions need a Color not a pointer to one

**Comment by raysan5** [comment]
@SusgUY446 I was not referring to a pointer, I was referring to `ColorName()` functions group, for naming consistency.

**Comment by SusgUY446** [comment]
oh okay.

File: src/raylib.h [hunk 1]
```diff
@@ -1425,6 +1425,7 @@ RLAPI int ColorToInt(Color color);                                          // G
 RLAPI Vector4 ColorNormalize(Color color);                                  // Get Color normalized as float [0..1]
 RLAPI Color ColorFromNormalized(Vector4 normalized);                        // Get Color from normalized values [0..1]
 RLAPI Vector3 ColorToHSV(Color color);                                      // Get HSV values for a Color, hue [0..360], saturation/value [0..1]
+RLAPI Color ColorLerp(Color color1, Color color2, float d);                 // Mix 2 Colors together
```
Comment (raysan5) [inline src/raylib.h:1428]: Please, could you move this function to the last position of Color() functions? Also, review the comment to follow consistency with other functions.

File: src/rtextures.c [hunk 2]
```diff
@@ -4985,6 +4985,27 @@ Vector3 ColorToHSV(Color color)
     return hsv;
 }
 
+/*
```
Comment (raysan5) [inline src/rtextures.c:4988]: Please, could you review the comment to be consistent with the other color functions. No functions use that many comment lines and neither /* */ comments.

**Comment by raysan5** [comment]
@SusgUY446 Please, could you review the mentioned points?

**Comment by SusgUY446** [comment]
okay

**Comment by raysan5** [comment]
@SusgUY446 Thanks for the improvement, I'm reviewing it...

**Pull request merged**